pkg/velerorunner/task.go

```diff
@@ -230,11 +230,11 @@ func (t *Task) updateBackup(backup *velero.Backup) error {
 	backup.Spec = velero.BackupSpec{
 		StorageLocation:         backupLocation.Name,
 		VolumeSnapshotLocations: []string{"aws-default"},
-		TTL:                metav1.Duration{Duration: 720 * time.Hour},
-		IncludedNamespaces: namespaces,
-		ExcludedNamespaces: []string{},
-		IncludedResources:  t.BackupResources,
-		ExcludedResources:  []string{},
+		TTL:                     metav1.Duration{Duration: 720 * time.Hour},
```

Comment:
Changed by `fmt`.

Reply:
Everyone's `fmt` seems to be battling it out